webpack.config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         use: ['style-loader', 'css-loader', 'sass-loader']
       },
       {
-        test: /\.(ttf|otf|eot|svg|woff(2)?)(\?[a-z0-9]+)?$/,
+        test: /\.(ttf|otf|eot|woff(2)?)(\?[a-z0-9]+)?$/,
```

Comment:
This was picking up .svg files and putting them in a folder named `fonts`, which was confusing, so I just removed this because we don't have any custom fonts anyway.